app-emulation/wine-proton/wine-proton-7.0.6.ebuild

```
@@ -178,6 +178,9 @@ src_prepare() {
 	eautoreconf
 	tools/make_requests || die # perl
 	dlls/winevulkan/make_vulkan -x vk.xml || die # python, needed for proton's
+	# tip: if need more for user patches, with portage can e.g. do
+    # echo "post_src_prepare() { tools/make_specfiles || die; }" \
+    #     > /etc/portage/env/app-emulation/wine-proton
```

```suggestion
	# echo "post_src_prepare() { tools/make_specfiles || die; }" \
	#     > /etc/portage/env/app-emulation/wine-proton
```
(same for the others, has whitespaces before `#`)